### PR TITLE
SONAR-9616 Write main branch to DB even if plugin is not installed

### PR DESCRIPTION
**Pull request by dbmeneses**

### CODE REVIEW

File: server/sonar-server/src/main/java/org/sonar/server/projectbranch/ws/BranchDtoToWsBranch.java [hunk 1]
```diff
@@ -45,14 +44,13 @@ private BranchDtoToWsBranch() {
   static WsBranches.Branch.Builder toBranchBuilder(BranchDto branch, Optional<BranchDto> mergeBranch, Map<String, MeasureDto> measuresByMetricKey) {
```
Comment (SonarTech) [inline server/sonar-server/src/main/java/org/sonar/server/projectbranch/ws/BranchDtoToWsBranch.java]: MINOR Specify a "BranchDto" parameter instead. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS3553)

**Comment by SonarTech** [comment]
SonarQube analysis reported 1 issue
* ![MINOR][MINOR] 1 minor

Watch the comments in this conversation to review them.

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: server/sonar-server/src/main/java/org/sonar/server/computation/task/projectanalysis/component/BranchLoader.java [hunk 2]
```diff
@@ -27,7 +27,7 @@
 import static org.apache.commons.lang.StringUtils.trimToNull;
 
 public class BranchLoader {
-
+  public static final String DEFAULT_MAIN_BRANCH_NAME = "master";
```
Comment (julienlancelot) [inline server/sonar-server/src/main/java/org/sonar/server/computation/task/projectanalysis/component/BranchLoader.java]: I think this constant would deserve to be in BranchDto

Comment (dbmeneses) [reply server/sonar-server/src/main/java/org/sonar/server/computation/task/projectanalysis/component/BranchLoader.java]: ok

**Pull request merged**